main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import(
+	"github.com/julienschmidt/httprouter"
+	"log"
+	"net/http"
+)
+
+func Home(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	w.WriteHeader(http.StatusOK)
+}
+
+func addRoutes(router *httprouter.Router) {
```

Comment:
Did you consider a signature like this?

```go
func newRouter() *httprouter.Router
```

Where it returns a router fully-configured for the application?

It could even be like this, to encourage acceptance-level testing:

```go
func newRouter() http.Handler
```